packages/new-polymath-issuer/src/pages/DividendsWizard/Step-3/index.tsx

```diff
@@ -255,18 +255,36 @@ const Step3Base: FC<Props> = ({
         }}
         validate={handleValidation}
         onSubmit={onSubmit}
-        render={({ values }) => {
+        render={({
+          values,
+          setFieldTouched,
+          isValid,
+          initialValues,
+          touched,
+        }) => {
           const { currency } = values;
-
           return (
             <form
               onSubmit={submitEvent => {
```

Comment:
Lets move this to a handler function instead of having this inline. If you need some of the values in the render context pass them as arguments